src/c20_client/retrieve_docket.py

```diff
@@ -3,19 +3,22 @@
 """
 import requests
 from c20_client.status_code_check import check_status
+from c20_client.client_logger import LOGGER
 
 
 def get_docket_data(api_key, docket_id):
     """
     Makes call to regulations.gov and retrieves the docket data
     """
+    LOGGER.info('Requesting docket from regulations.gov')
     response = requests.get("https://api.data.gov:443/" +
                             "regulations/v3/docket.json?api_key=" +
                             api_key +
                             "&docketId=" +
                             docket_id)
 
     check_status(response.status_code)
+    LOGGER.info('document has been retrieved')
```

Comment:
I believe  here the massage should be ('docket has been retrieved')

Reply:
the typo has been fixed